README.md

```diff
@@ -1,2 +1,48 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
 # Awsome-Books-ES6
-Create an application to add and remove books using JavaScript ES6
+
+In this project, we will build a basic website that allows users to add/remove books from a list using JavaScript ES6 and Modules 
+
+## Built With
+
+- Javascript, HTML and CSS
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps:
+- Git clone the repository to your local machine:
+```
+https://github.com/AlejandraKeber/Awsome-Books-ES6.git
+```
+- Open the folder with you code editor
+
+
+## Authors:
+
+👤 **Alejandra Keber Maggi:** 
+
+- GitHub: [@AlejandraKeber](https://github.com/AlejandraKeber)
+- Twitter: @KeberAlejandra
+- LinkedIn: [alejandra-keber](www.linkedin.com/in/alejandra-keber)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Hat tip to anyone whose code was used
+- Inspiration
+- etc
+
+## 📝 License
+
+This project is [MIT](./MIT.md) licensed.
```

Comment:
- [ ] The Link to MIT.md file is broken, So I suggest adding the `MIT.md` file to your project 👌, Here is the [Micorverse Template](https://github.com/microverseinc/readme-template/blob/master/MIT.md), and remember to customize it 😉

> The MIT license gives users express permission to reuse code for any purpose, sometimes even if the code is part of proprietary software. As long as users include the original copy of the MIT license in their distribution, they can make any changes or modifications to the code to suit their own needs. 💯